Simplified/SignInLogic/NYPLSignInBusinessLogic+CardCreation.swift

```diff
@@ -9,10 +9,115 @@
 import Foundation
 import NYPLCardCreator
 
-extension JuvenileFlowCoordinator: JuvenileFlowCoordinating {}
+extension FlowCoordinator: FlowCoordinating {}
 
 extension NYPLSignInBusinessLogic {
+  /// The entry point to the regular card creation flow.
+  /// - Note: This is available only for NYPL accounts.
+  /// - Parameters:
+  ///   - completion: Always called whether the library supports
+  ///   card creation or not. If it's possible, the handler returns
+  ///   a navigation controller containing the VCs for the whole flow.
+  ///   All the client has to do is to present this navigation controller
+  ///   in whatever way it sees fit.
+  @objc
+  func startRegularCardCreation(
```

Comment:
this method is very long, I am wondering if it can be split into smaller functions. Without doing the work (I can do it in the following days) do you think that's possible?

Reply:
Maybe we can put the completionHandler on line 88 into a function like `private func regularFlowCompletionHandler() -> ((_ username: String, _ PIN: String, _ userInitiated: Bool) -> Void)`. What do you think?

Reply:
that makes sense…. anyway don’t worry about doing that, thanks for the suggestion! hopefully i’ll do it soon or we can always do it at a later date. The flow did seem easy to understand to me despite the function length, which is what’s most important.